backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComposeComponentUsageDetector.kt

```diff
@@ -159,6 +159,20 @@ class BpkComposeComponentUsageDetector : Detector(), SourceCodeScanner {
         "androidx.compose.material.TextField",
       )
     ),
+    RANGE_SLIDER(
+      fullName = "net.skyscanner.backpack.compose.slider.BpkSlider",
```

Comment:
this should be `BpkRangeSlider` as the function that replaces it is that - the file it is in isn't really important for consumers